src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomOoyala.java

```diff
@@ -15,8 +15,11 @@
 
 import java.awt.*;
 
+@Test(groups = "AdsFandomOoyala")
 public class TestAdsFandomOoyala extends AdsFandomTestTemplate {
   private static final String PLAY_BUTTON_SELECTOR = ".ooyala-video .oo-action-icon";
+  private static final String PLAYER_CONTAINER_SELECTOR
```

Comment:
Please, define here By.cssSelector except String, it will be cleaner and you have it in name. If it is easy consider refactor rest of constants